src/ocm/components/clusterWizard/CustomManifestStep.tsx

```diff
@@ -17,7 +17,7 @@ const getInitialFormStateProps = () => {
     isEmpty: true,
   };
 };
-const CustomManifestStep: React.FC<{ cluster: Cluster }> = ({ cluster }) => {
+const CustomManifestStep = ({ cluster }: { cluster: Cluster }) => {
```

Comment:
What is wrong about using `React.FC`?

Reply:
https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/

Reply:
I remember that `React.FC` helped me to avoid bugs by guarding the return type when modifying more complex components (I simply overlooked a branch for `return`). So I suggest using `JSX.Element` as a return type.
By doing that, it's questionable if avoiding `React.FC` brings any benefit except the fact that it is marked as deprecated in React 18.

Reply:
ok